packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.tsx

```diff
@@ -262,7 +265,7 @@ export const ChatbotConversationHistoryNav: FunctionComponent<ChatbotConversatio
           >
             <DrawerCloseButton onClick={onDrawerToggle} {...drawerCloseButtonProps} />
             {onNewChat && (
-              <Button size={isCompact ? 'sm' : undefined} onClick={onNewChat}>
+              <Button size={isCompact ? 'sm' : undefined} onClick={onNewChat} isDisabled={newChatButtonDisabled}>
```

Comment:
Would you be willing to make this more generic?

For example:

```suggestion
              <Button size={isCompact ? 'sm' : undefined} onClick={onNewChat} {...newChatButtonProps}>
```

And then in your demo, you would pass in `newChatButtonProps={{isDisabled: true}}`. This would let us offer you even more of the regular control you'd expect over buttons without new PRs. I think it was an oversight that we have this locked down so much. Nicole is very much in favor of giving people control like this.